Assets/BossRoom/Scripts/Client/UI/PartyHUD.cs

```diff
@@ -158,6 +161,14 @@ void SetUIFromSlotData(int slot, NetworkCharacterState netState)
             m_PartyHealthSliders[slot].value = netState.HitPoints;
             m_PartyNames[slot].text = GetPlayerName(netState);
 
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+            m_PartyHealthGodModeImages[slot].gameObject.SetActive(netState.NetworkLifeState.IsGodMode.Value);
+            netState.NetworkLifeState.IsGodMode.OnValueChanged += (value, newValue) =>
+            {
+                m_PartyHealthGodModeImages[slot].gameObject.SetActive(newValue);
+            };
+#endif
+
```

Comment:
For sanity's sake, include remove the subscription to this callback when the hero or allies are removed.